content/docs/contributing/building.md

```diff
@@ -3,123 +3,211 @@ title: Building cert-manager
 description: 'cert-manager contributing guide: Building cert-manager'
 ---
 
-cert-manager makes use of [Bazel](https://bazel.build/) to build the project. 
-Bazel manages all developer dependencies, Helm chart building, Docker images and the code itself. 
-We try to use it as much as possible.
-We currently use Bazel `v4.2.1`. The minimum supported version is `v4.0.0`.
+cert-manager is built and tested using [make](https://www.gnu.org/software/make/manual/make.html), with a focus on using the standard Go tooling
+where possible and keeping system dependencies to a minimum. The cert-manager build system can provision most of its dependencies - including Go -
+automatically if required.
 
-> **TIP**: are you using GoLand? Make sure to exclude the `bazel-` folders! You can do this by right clicking on the folder -> Mark Directory As -> Excluded
-> This will save you a ton of CPU time!
+cert-manager's build system fully supports developers who use `Linux amd64`, `macOS amd64` and `macOS arm64`. Other operating systems and architectures may
+work, but are largely untested.
 
-> **TIP**: are you sitting on a corporate network with internal PKI? Bazel does not honor custom CA certificates by default, but depending on your OS
-> a [clean workaround might be available](https://groups.google.com/g/bazel-discuss/c/13uPDObyfQg/m/UjPbalztCQAJ).
+## Prerequisites
 
-## A quick intro to Bazel
+There are very few other requirements needed for developing cert-manager, and crucially the build system should tell you with a friendly error
+message if there's anything missing. If you think an error message which relates to a missing dependency is unhelpful, we consider that a bug and
+we'd appreciate if you raised [an issue](https://github.com/cert-manager/cert-manager/issues/new?assignees=&labels=&template=bug.md) to tell us about it!
 
-Bazel has 3 main commands which we use:
+You should install the following tools before you start developing cert-manager:
 
-`bazel build [...]` will build and compile code for you e.g. `bazel build //cmd/ctl` will build our CLI. 
-`bazel test [...]` will run any tests for a given package
-`bazel run [...]` is only used to run certain scripts not the compiled code (unlike Go). e.g. `bazel run //hack/bin:helm` will download and run Helm.
+- [git](https://git-scm.com/)
+- [curl](https://curl.se/)
+- GNU Coreutils (available via [homebrew](https://formulae.brew.sh/formula/coreutils) for macOS)
+- `jq` (available in package managers and in homebrew)
+- `docker` (or `podman`, see [Container Engines](#container-engines) below)
```

Comment:
Suggestion: if by default all targets use system's Go I'd add Go here as well- it feels more straightforward

Reply:
Good shout, yeah, I'll add :+1: